beancount.el

```diff
@@ -232,14 +239,9 @@ from the open directive for the relevant account."
 
 (defun beancount-outline-face ()
   (if outline-minor-mode
-      (cl-case (funcall outline-level)
-      (1 'org-level-1)
-      (2 'org-level-2)
-      (3 'org-level-3)
-      (4 'org-level-4)
-      (5 'org-level-5)
-      (6 'org-level-6)
-      (otherwise nil))
+      (let ((level (funcall outline-level)))
+        (if (and (integerp level) (<= 1 level 8))
+            (intern (format "beancount-level-%d" level))))
```

Comment:
What is the point of this? I think the `cl-case` version is much easier to read and a couple of order of magnitude more efficient. I would prefer to leave this as it is.

Reply:
I don't think it's difficult to read, but I can expand it back into a `cl-case` if you prefer.

Reply:
The `cl-case` is stupidly obvious but I cannot say the same about this incantation (and I am sure the `(integerp level)` check didn't come natural but required trying and debugging). Please revert back to the `cl-case` using `outline-n` instead of `org-level-n`.

Reply:
> I am sure the (integerp level) check didn't come natural but required trying and debugging). 

Not at all, but I am happy to agree to disagree. Thanks for your time!